casper/src/main/scala/coop/rchain/casper/util/ProtoUtil.scala

```diff
@@ -517,36 +507,19 @@ object ProtoUtil {
       phloLimit = phlos
     )
 
+  def sourceDeployNow(source: String): DeployData =
+    sourceDeploy(
+      source,
+      System.currentTimeMillis(),
+      accounting.MAX_VALUE
+    )
+
   def compiledSourceDeploy(
```

Comment:
I think we should inline this to not cause the impression that the compilation process is avoided downstream.